### DISCUSSION TITLE
Training Webinar

### DISCUSSION THREAD

**Discussion by SWi1** [SchlossLab/mikropml#307] [Ideas]
Hello, I attended Pat Schloss' Dec. workshop on mothur and through that, learned about the mikropml package. I'm very excited about this new package and to try it out with my data. Is there a possibility that one or a few of the package creators would be open to hosting a mikropml training webinar? 

Thank you for putting this package together!

Stephanie

**Comment by BTopcuoglu** [comment]
Hi @SWi1,
Thank you for reaching out and using our new package! We’re trying to pull something together to host a `mikropml` training webinar. Keep an eye on this space for news and updates :)

Happy New Year!
-B.

**Comment by pschloss** [comment]
Hi @SWi1 - in case you didn't see it, I have been populating a [playlist on the Riffomonas YouTube channel](https://www.youtube.com/playlist?list=PLmNrK_nkqBpKpzb9-vI4V7SdXC-jXEcmg) that is going through mikropml.
